docs/sources/patterns/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+aliases:
```

Comment:
If the page has never existed before you don't need an `alias`, it is to create redirects when a page is renamed or moved.